drivers/android/process.rs

```diff
@@ -366,7 +366,7 @@ impl Process {
         self.inner.lock().push_work(work)
     }
 
-    fn set_as_manager(&self, info: Option<FlatBinderObject>, thread: &Thread) -> Result {
+    fn set_as_manager(self: &Ref<Self>, info: Option<FlatBinderObject>, thread: &Thread) -> Result {
```

Comment:
Why take `self: &Ref<Self>` instead of `&self`? It isn't cloned anywhere in this function and thus should be functionally identical to `&self`, right?

Reply:
Never mind. I see that `self.get_node` does clone it sometimes.

Reply:
it calls `self.get_node()`, which calls `self.clone()`.